packages/workspace/src/browser/workspace-service.ts

```diff
@@ -179,6 +179,24 @@ export class WorkspaceService implements FrontendApplicationContribution {
         return options !== undefined && !!options.preserveWindow;
     }
 
+    /**
+     * Return true if one of the paths in paths array is present in the workspace
+     */
+    async contains(paths: string[]): Promise<boolean> {
```

Comment:
I would add a few more remarks here:
 - you do not need `Promise.resolve(false)` here, just `false`, for instance. (Please review other `async` methods in this module.)
 - can we name this `containsSome` instead of `contains`.
 - this `paths` can be tricky on different platforms, I would add this to the documentation. Something like you need to use `/` for the paths.